src/VipsTargetCustom.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Jcupitt\Vips;
+
+use Closure;
+
+class VipsTargetCustom extends VipsTarget
+{
+    /**
+     * A pointer to the underlying VipsTargetCustom. This is the same as the
+     * GObject, just cast to VipsTargetCustom to help FFI.
+     *
+     * @internal
+     */
+    public \FFI\CData $pointer;
+
+    public function __construct()
+    {
+        $this->pointer = FFI::vips()->vips_target_custom_new();
+        parent::__construct($this->pointer);
+    }
+
+    /**
+     * Attach a write handler.
+     * The interface is exactly as fwrite. The handler is given a bytes-like object to write,
+     * and should return the number of bytes written.
+     * @throws Exception
+     */
+    public function onWrite(Closure $callback): void
```

Comment:
I didn't actually know about the difference. I don't think it would change anything for the FFI part, so I'll see if I can do some work on this PR next week to get some changes in/open a new one if John's quicker than me.

Reply:
The difference is that `Closure` must be an anonymous function, where `callable` also can be a normal function.